### DISCUSSION TITLE
Specter Desktop hangs indefinitely on "Launching Specter Desktop" step during startup

### DISCUSSION THREAD

**Discussion by phjlljp** [cryptoadvance/specter-desktop#823] [Help]
I'm trying to get Specter Desktop setup with my local Bitcoin Core Node, but as the title suggests, I can't get Specter to boot reliably across multiple operating systems. I initially tried to run it on the same computer as my node (running OS X High Sierra) but it just hung  indefinitely on "Launching Specter Desktop", next I tried running it on a different machine running Ubuntu (20.04 LTS), intending to connect using RPC. Initially, I was able to boot to main page, but couldn't get JSON-RPC to work. Before I could start debugging JSON-RPC though, that installation started hanging as well on startup. Next, I tried installing Specter on third machine, running OS X Catalina, but experienced the exact same issue as on the previous two machines: hanging on "Launching Specter Desktop" step during startup. If I wait 10m, it will eventually load the main page, but none of the buttons/sections will load (it hangs forever).

I have no idea what's going on, but would greatly appreciate any ideas/thoughts.

**Comment by ben-kaufman** [comment]
Hi, the desktop app currently supports macOS Catalina (10.15) or above. You will need to [install Specter from Pip](https://github.com/cryptoadvance/specter-desktop/tree/v0.10.4#installing-specter-from-pip) to use it on lower macOS versions.
We hope to support older macOS versions soon though and will update accordingly when that's avaialble.
